vespa-http-client/src/test/java/com/yahoo/vespa/http/client/core/operationProcessor/OperationProcessorTest.java

```diff
@@ -410,4 +412,18 @@ public void testSendsResponseToQueuedDocumentOnClose() throws InterruptedExcepti
         operationProcessor.close();
         countDownLatch.await();
     }
+
+    @Test
+    public void testUnknownHost() {
+        try {
+            new SessionParams.Builder()
+                    .addCluster(new Cluster.Builder().addEndpoint(Endpoint.create("localhost")).build())
+                    .addCluster(new Cluster.Builder().addEndpoint(Endpoint.create("unknown")).build())
+                    .build();
+        }
+        catch (IllegalArgumentException e) {
```

Comment:
Consider using an exception expectation as part of the test declaration instead of an explicit `catch`. This test will currently succeed if the code doesn't throw as well.

Reply:
But I want to assert on the error message.